airmozilla/new/templates/new/_email_new_event.html

```diff
@@ -19,6 +19,16 @@
   >{{ build_absolute_url(url('main:event', event.slug)) }}</a>
 </p>
 
+{% if event.channels.all().exists() %}
+<p>
+  Channels:
+  {% for channel in event.channels.all() %}
+    <a href="{{ build_absolute_url(url('main:home_channels', channel.slug)) }}">{{ channel }}</a>
+    {% if not loop.last %}, {% endif %}
```

Comment:
Move this `if` to be right after the `</a>` on the same line. 
Otherwise you get this HTML output:
```html
<p>
  Channels:
    <a href="...">Channel1</a> ,
    <a href="...">Channel2</a> , 
    <a href="...">Channel3</a>
<p>
```